cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,29 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+/// <reference types='cypress' />
```

Comment:
The reference to Cypress types is duplicated. Remove the redundant reference to avoid confusion.